tasks/remote_storage.py

```diff
@@ -176,6 +182,7 @@ def create_metadata_json(src, uuid, repo_owner, pr_number, pr_author,
         'task_name': task_name,
         'returncode': returncode,
         'mtime': datetime.now().strftime('%Y-%m-%d %H:%M'),
+        'artifacts': artifacts,
```

Comment:
Calling `json.dump` raises:
```python
*** TypeError: Object of type 'datetime' is not JSON serializable
```
```python
# artifacts data:
[
    {'name': '.vagrant', 'mtime': datetime.datetime(2019, 3, 29, 17, 0, 18, 457159), 'size': 42, 'type': 'dir'}, 
    {'name': 'rpms', 'mtime': datetime.datetime(2019, 3, 29, 17, 8, 14, 687826), 'size': 4096, 'type': 'dir'}, 
    #...
}
```
You probably need to call `.isoformat()` for key `mtime` on every entry.